docs/quickstart.rst

```diff
@@ -50,27 +50,30 @@ the video directive supports all the optional attributes from the html tag as su
     ``:alt:``,``str``,Specify the text to write when the video cannot be displayed
     ``:autoplay:``,,Specifies that the video will start playing as soon as it is ready
     ``:nocontrols:``,,Specifies that video controls should not be displayed (such as a play/pause button etc).
-    ``:height:``,``int``,Sets the height of the video player in pixels
+    ``:height:``,``int``,Sets the height of the video player in pixels (ignored if relative width is used)
     ``:loop:``,,Specifies that the video will start over again, every time it is finished
     ``:muted:``,,Specifies that the audio output of the video should be muted
     ``:poster:``,``str``, Specifies an image url to be shown while the video is downloading, or until the user hits the play button
     ``:preload:``,``str``,"Specifies if and how the author thinks the video should be loaded when the page loads. Can only be values from ``['auto', 'metadata', 'none']``"
-    ``:width:``,``int``, Sets the width of the video player in pixels
+    ``:width:``,``int``\ [``%``\ ], Sets the width of the video player in pixels or relative to the page's width if a percentage
 
 They can be used as any directive option:
 
 .. code-block:: rst
 
     .. video:: _static/video.mp4
         :nocontrols:
+        :autoplay:
+        :muted:
         :loop:
-        :poster: _static/image.png
```

Comment:
why did you removed the poster image ?

Reply:
It was in the code block but not in the actual `videp` directive.

Reply:
it was at least so let's keep it for now and open a dedicated issue if it's never used.